test/icon.cxx

```diff
@@ -23,7 +23,7 @@ static Fl_Double_Window *win;
 
 void choice_cb(Fl_Widget *, void *v) {
   Fl_Color c = (Fl_Color)(fl_intptr_t)v;
-  uchar buffer[32*32*3];
+  uchar buffer[32*32*3] = {};
```

Comment:
Fixed in 1.4.0 by making this variable 'static' which fixes the warning w/o an initializer (because static variables are initialized with zeroes anyway).

Reply:
Backported from 1.4.0